src/main/java/Game.java

```diff
@@ -0,0 +1,54 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Game {
+
+    List<Car> carList = new ArrayList<Car>();
+    List<Car> winnerList = new ArrayList<Car>();
+
+    public void playGame(int carNum, int roundNum) {
+        initGame(carNum);
+        for (int i = 0; i < roundNum; i++) {
+            playRound();
+        }
+        makeWinnerList();
+    }
+
+    public void initGame(int carNum){
+        for (int i = 0 ; i < carNum ; i++)
+            carList.add(new Car("Car_"+i+""));
+    }
+
+    public void playRound() {
+        for (Car car : carList) {
+            car.move(getRandom());
+        }
+    }
+
+    public int getRandom(){
+        return (int)(Math.random()*10);
+    }
+
+    public void makeWinnerList() {
+        Car winner = carList.get(0);
+        int maxDistance = getMaxDistance();
+        for (Car car : carList) {
+            addWinner(car, maxDistance);
+        }
+    }
+
+    private void addWinner(Car car, int maxDistance) {
+        if (car.getDistance() == maxDistance) {
+            winnerList.add(car);
+        }
```

Comment:
car의 position 값을 꺼내 값을 비교하는 것보단, car에서 해결하면 어떨까요?
```suggestion
        if (car.isSamePosition(maxDistance)) {
            winnerList.add(car);
        }
```
`Car`에게 메세지를 전달하는 방식으로 개선 할 수 있을 것 같아요!